README.md

```diff
@@ -238,6 +239,7 @@ The [To C Or Not To C](https://gist.github.com/ISSOtm/4f4d335c3fd258ad0dfc7d4d61
 - [Gambatte](https://github.com/sinamas/gambatte) - Cross-platform and accurate emulator.
 - [MetroBoy](https://github.com/aappleby/MetroBoy) - A playable, circuit-level simulation of an entire Game Boy.
 - [gbe-plus](https://github.com/shonumi/gbe-plus) - A recently rewritten emulator that has a large effort in preserving the functions of obscure accessories (such as IR link, Mobile Network GB, Barcode Boy, GB Printer, local and online GB Serial Link Cable, ... )
+- [Emulicious](https://emulicious.net/) - Emulator with useful development tools such as source level debugging for ASM and C in VS Code.([VS Code debug adapter](https://marketplace.visualstudio.com/items?itemName=emulicious.emulicious-debugger))

```

Comment:
Don't forget a space after the period; I would also rephrase this in a more condensed manner:
```markdown
Includes source-level debugging for ASM and C via a [VS Code debug adapter](https://marketplace.visualstudio.com/items?itemName=emulicious.emulicious-debugger).
```

Reply:
:heavy_check_mark: